### DISCUSSION TITLE
docs/3.x/developer/extending/apps/manifest

### DISCUSSION THREAD

**Discussion by giscus** [saleor/saleor-docs#781] [General]
# docs/3.x/developer/extending/apps/manifest

The App manifest is a JSON file that describes the application, including its name, version, required permissions, and how it handles the store's data.

https://docs.saleor.io/docs/3.x/developer/extending/apps/manifest

**Comment by JannikZed** [comment]
Defining webhooks in the manifest file is not very consistent. The AppManifest type says for the field "query": `If query is not provided, the default webhook payload will be used` 
But when trying to install an app without "query" defined in our Saleor 3.13, we receive the following error: 
```
Something went wrong
Missing required fields for webhook: query.
```
Please be consistent! The query feature is really nice, but this field should be optional to make it more easy to just getting started with defaults.

**Reply by lkostrowski** [reply]
https://github.com/saleor/saleor-app-sdk/pull/280
